elastic/kibana
# [Security Solution][Entity Analytics] Show missing risk score privileges in entity risk tabs

## Summary

This PR adds a check against the risk scoring privileges and shows the appropriate message in case they are missing.
Previously, in the entity pages, if a user did not have the correct privileges, clicking on the risk tab would show a message to "Enable risk scoring". This was incorrect as the risk engine could be enabled but the user might not have the permissions to view risk scores.

Previously, in **Hosts/Users page -> Host risk / User risk tab**
![image](https://github.com/elastic/kibana/assets/2423976/0d3f8073-0e5b-4e71-91b0-7d8e3fd5357c)

Now:
![Screenshot 2024-04-16 at 17 41 34](https://github.com/elastic/kibana/assets/2423976/304e8af9-54af-4acf-9037-58316734bd15)


## How to test

1. Create a role which has access to all security related indices and Security Permissions.
2. Ensure that the user does not have permissions to view the `.risk-score*` index
3. Create a new user with said role from step 1
4. Login with that user and navigate to **Hosts/Users page -> Host risk / User risk tab**
